src/components/goalDetail/GoalHeader.tsx

```
@@ -1,31 +1,100 @@
-import { useGoals } from '@/hooks/useGoals';
+import { useGoals, useUpdateGoal, useDeleteGoal } from '@/hooks/useGoals';
 import GoalProgress from './GoalProgress';
 import { Goal } from '@/types/goals';
 
-interface Props {
+type Props = {
   doneItems: number;
   todoItems: number;
   id: string;
-}
+  setGoalAvailable: (value: boolean) => void;
```

ESLint

[error] 'value' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)